### PR TITLE
GOV 287 Enabled liveness and readiness for the key auth story

### PR DESCRIPTION
**Pull request by danishjamal104**
###

### CODE REVIEW

**Review by fynmanoj — APPROVED** [review]

File: src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java [hunk 1]
```diff
@@ -37,19 +40,23 @@ public class BatchTransactionsController implements BatchTransactions {
...
                 .addHeader(PURPOSE,purpose)
                 .addHeader(FILE_NAME,localFileName)
                 .addHeader("Type",type)
-                .addHeader("Platform-TenantId",tenant)
+                .addHeader(HEADER_PLATFORM_TENANT_ID,tenant)
+                .addHeader(HEADER_REGISTERING_INSTITUTE_ID, registeringInstitutionId)
+                .addHeader(HEADER_PROGRAM_ID, programId)
                 .build();
-
+		log.info("Headers passed: {}", headers);
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java:56]: use debug logs

File: src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java [hunk 2]
```diff
@@ -37,19 +40,23 @@ public class BatchTransactionsController implements BatchTransactions {
...
         Exchange exchange = SpringWrapperUtil.getDefaultWrappedExchange(producerTemplate.getCamelContext(),
               headers);
+        log.info("Header in exchange: {}", exchange.getIn().getHeaders());
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java:59]: use debug logs

File: src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java [hunk 3]
```diff
@@ -285,12 +254,14 @@ private void setup() {
...
                     variables.put(NOTE, note);
+                    variables.put(CLIENT_CORRELATION_ID, exchange.getProperty(CLIENT_CORRELATION_ID));
                     variables.put(PROGRAM_NAME, exchange.getProperty(PROGRAM_NAME));
                     variables.put(PAYER_IDENTIFIER_TYPE, exchange.getProperty(PAYER_IDENTIFIER_TYPE));
                     variables.put(PAYER_IDENTIFIER_VALUE, exchange.getProperty(PAYER_IDENTIFIER_VALUE));
                     variables.put(REGISTERING_INSTITUTE_ID, exchange.getProperty(REGISTERING_INSTITUTE_ID));
+                    variables.put(IS_FILE_VALID, true);
                     setConfigProperties(variables);
-
+					logger.info("Zeebe variables published: {}", variables);
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java:264]: use debug logs

File: src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java [hunk 4]
```diff
@@ -188,7 +157,7 @@ private void setup() {
                     logger.info("Inst id: {}, prog id: {}", registeringInstituteId, programId);
                     if (!(StringUtils.hasText(registeringInstituteId) && StringUtils.hasText(programId))) {
                         // this will make sure the file is not updated since there is no update in data
-                        logger.info("Reh or pro is null");
+                        logger.info("InstitutionId or programId is null");
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java:160]: use debug logs

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java (hunk 3 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java]: done

File: src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java (hunk 2 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java]: done

File: src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java (hunk 4 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java]: done

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**